ooni/geoip.py

```diff
@@ -164,8 +164,10 @@ class ProbeIP(object):
 
     def __init__(self):
         self.geoIPServices = {
-            'ubuntu': UbuntuGeoIP,
-            'duckduckgo': DuckDuckGoGeoIP
+            'ubuntu': UbuntuGeoIP
+            # We are disabling this because it sometimes creates parsing
+            # errors.
+            # 'duckduckgo': DuckDuckGoGeoIP
```

Comment:
Needs xref to an issue where we fix lack of "redundancy" for the check.

Here is my `myip` script with some samples:
```
# Google Global Cache Installation and Operations Guide / February 17, 2015
# https://peering.google.com/static/downloads/GGCInstallation-R720.pdf
dig @ns1.google.com. -t txt o-o.myaddr.l.google.com. +short | tr -d '"'
dig @ze.akamaitech.net. -t A whoami.akamai.net. +short
# resolver1.opendns.com & resolver2.opendns.com
dig @208.67.222.222 -t A myip.opendns.com. +short
{ curl -sS -4 https://diagnostic.opendns.com/myip; echo; } | cat
{ curl -sS -4 https://yandex.com/internet/api/v0/ip; echo; } | tr -d '"'
```